### PR TITLE
Refactor/card how we do

### PR DESCRIPTION
**Pull request by ramonxm**
## Description

Ajustes de estilo

## Developer Checks

- [x] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [x] Tested on browser device
- [x] Tested on Mobile device

### CODE REVIEW

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.81 | ✅ |
| First Contentful Paint | 2.5 s |  |
| Time to Interactive | 4.7 s |  |
| Speed Index | 2.8 s |  |
| Total Blocking Time | 360 ms |  |
| Largest Contentful Paint | 3.5 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.94 | ✅ |
| First Contentful Paint | 1.8 s |  |
| Time to Interactive | 5.7 s |  |
| Speed Index | 2.6 s |  |
| Total Blocking Time | 130 ms |  |
| Largest Contentful Paint | 1.8 s |  |
| Cumulative Layout Shift | 0.044 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: src/components/Courses/styles.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ type StyleProps = {
   width?: string;
 };
 
-const CourseContainer = styled.section`
+export const CourseContainer = styled.section`
```
Comment (jonataspinto) [inline src/components/Courses/styles.ts:8]: Padrão é tudo 💚

File: src/components/StudentCharacteristicsCard/styles.ts [hunk 2]
```diff
@@ -14,8 +21,30 @@ export const CardContainer = styled.div<{ variation: 'right' | 'left' }>`
   box-shadow: 0.3rem 0.3rem 0.5rem -0.3rem rgb(0 0 0 / 98%),
     ${({ theme }) =>
       `-0.1rem -0.1rem 0.3rem -0.3rem ${theme.palette.design.white}`};
-  align-self: ${({ variation }) =>
-    variation === 'right' ? 'flex-end' : 'flex-start'};
+
+  &:nth-child(even) {
```
Comment (jonataspinto) [inline src/components/StudentCharacteristicsCard/styles.ts:25]: 💚  💜

File: src/components/StudentCharacteristicsCard/styles.ts [hunk 3]
```diff
@@ -24,36 +53,8 @@ export const TextCharacteristics = styled(Typography).attrs({
   font-weight: 400;
 `;
 
-export const Division = styled.div`
-  background-color: ${({ theme }) => theme.palette.gray[3]};
+export const Division = styled.hr`
+  border-color: ${({ theme }) => theme.palette.gray[3]};
```
Comment (jonataspinto) [inline src/components/StudentCharacteristicsCard/styles.ts:57]: Ptuz, essa eu não vi

**Review by jonataspinto — APPROVED** [review]

File: src/components/StudentCharacteristicsCard/styles.ts (hunk 3 above)
Comment (ramonxm) [reply src/components/StudentCharacteristicsCard/styles.ts:57]: semantic

**Pull request merged**